src/NuGet.Core/NuGet.Commands/NuGet.Commands.csproj

```diff
@@ -4,10 +4,10 @@
 
   <PropertyGroup>
     <Description>Complete commands common to command-line and GUI NuGet clients.</Description>
-    <TargetFrameworks>$(TargetFrameworksLibrary)</TargetFrameworks>
+    <TargetFrameworks>$(TargetFrameworksLibrary);netstandard2.1</TargetFrameworks>
```

Comment:
Update the property in the common targets.

Reply:
Thanks! Changed accordingly.